tests/e2e/tests/e2e_happy_path/HappyPath.spec.ts

```diff
@@ -307,13 +298,13 @@ async function checkCodeNavigationWithContextMenu() {
     console.log('Known isuue https://github.com/eclipse/che/issues/14520.');
 }
 
-// sometimes under high loading the first click can be failed
-async function isureClickOnDebugMenu() {
-    try { await topMenu.selectOption('Debug', 'Open Configurations'); } catch (e) {
-        console.log(`After clicking to the Debug top menu the menu has been not opened, try to click again...`);
-        await topMenu.selectOption('Debug', 'Open Configurations');
-    }
-}
+// // sometimes under high loading the first click can be failed
```

Comment:
Do we really need to hold commented code here?

Reply:
Nope. This is wip.